client/blocks/image-editor/test/image-editor-canvas.jsx

```diff
@@ -1,38 +1,33 @@
+/** @jest-environment jsdom */
+jest.mock( 'blocks/image-editor/image-editor-crop', () => {
+	const { Component } = require( 'react' );
+
+	class ImageEditorCropMock extends Component {
+		constructor( props ) {
```

Comment:
I think I copied it over without too much thinking, I can revisit, thanks for pointing out.